core/base_service.go

```diff
@@ -38,6 +38,7 @@ import (
 const (
 	headerNameUserAgent = "User-Agent"
 	sdkName             = "ibm-go-sdk-core"
+	maxRedirects        = 10
```

Comment:
I'm not necessarily against making it configurable, but the net/http layer seems to hard-code it at 10 and I was trying to replicate the logic from the default "CheckRedirect" implementation.  I'm not aware of any requirement to make it configurable or to increase the max limit, and if we did go ahead and do that it would most likely be out of sync with the other transport layers.  So... we should proceed with caution IMO :joy:

Reply:
Oh, forget what I said! I didn't notice it when I checked the Go implementation, so yeah, it would make no sense to change it.. We should be in sync with the original implementation as much as possible.